### PR TITLE
Add test for save script using `Save-PSResource` without `-IncludeXml`

### PR DESCRIPTION
**Pull request by o-l-a-v**
# PR Summary

Add test for #1609

Which probably broke when fixing #1335

## PR Context

Add test "Save script without using -IncludeXML" for saving script using `Save-PSResource` without `-IncludeXml`.

Also fix typo and missing `-IncludeXml` in existing test "Save script using -IncludeXML".

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [x] None
    - **OR**
    - [ ] [Documentation needed]()
        - [ ] Issue filed: 
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] [Documentation needed]()
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [ ] N/A or can only be tested interactively
    - **OR**
    - [x] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
- **Tooling**
    - [x] I have considered the user experience from a tooling perspective and don't believe tooling will be impacted.
    - **OR**
    - [ ] I have considered the user experience from a tooling perspective and enumerated concerns in the summary.

### CODE REVIEW

**Comment by o-l-a-v** [comment]
@microsoft-github-policy-service agree

File: test/SavePSResourceTests/SavePSResourceV2Tests.ps1 [hunk 1]
```diff
@@ -173,10 +173,17 @@ Describe 'Test HTTP Save-PSResource for V2 Server Protocol' -tags 'CI' {
         $res.Version | Should -Be "1.0.0.0"
     }
 
+    It "Save script without using -IncludeXML" {
+        Save-PSResource -Name $testScriptName -Repository $PSGalleryName -Path $SaveDir -TrustRepository | Should -Not -Throw
+
+        $SavedScriptFile = [System.IO.Path]::Combine($SaveDir,('{0}.ps1' -f $testScriptName))
```
Comment (anamnavi) [inline test/SavePSResourceTests/SavePSResourceV2Tests.ps1:179]: Just a nitpick to use `Join-Path` to be consistent with the rest of our tests
```suggestion
        $SavedScriptFile = Join-Path -Path $SaveDir -ChildPath "$testScriptName.ps1"
```

File: test/SavePSResourceTests/SavePSResourceV2Tests.ps1 [hunk 2]
```diff
@@ -173,10 +173,17 @@ Describe 'Test HTTP Save-PSResource for V2 Server Protocol' -tags 'CI' {
...
+        [System.IO.File]::Exists($SavedScriptFile) | Should -BeTrue
```
Comment (anamnavi) [inline test/SavePSResourceTests/SavePSResourceV2Tests.ps1:180]: Just a nitpick to use powershell command `Test-Path` to be consistent with the rest of our tests
```suggestion
        Test-Path -Path $SavedScriptFile | Should -BeTrue
```

**Review by anamnavi — CHANGES_REQUESTED** [review]
This looks great, thanks for adding this test and correcting the regression of our older test. I've added a few suggestions, mostly nitpicks to use PowerShell commands as we do throughout our other tests, but once that's in I'll approve, thanks @o-l-a-v !

**Comment by o-l-a-v** [comment]
@anamnavi : @SydneyhSmith told me to put tests into #1614 in <https://github.com/PowerShell/PSResourceGet/pull/1614#issuecomment-2040349595>. So close this PR and continue in mentioned PR?

**Comment by anamnavi** [comment]
@o-l-a-v ah yes, I missed that earlier. I'll close this PR. And add the comments from this to that PR.

**Pull request closed without merging**